worker/dbaccessor/tracker_test.go

```diff
@@ -77,8 +74,7 @@ func (s *trackedDBWorkerSuite) TestWorkerDBIsNotNil(c *gc.C) {
 
 	w, err := s.newTrackedDBWorker(defaultPingDBFunc)
 	c.Assert(err, jc.ErrorIsNil)
-
-	defer workertest.CleanKill(c, w)
+	defer w.Kill()
```

Comment:
We should use dirty kill here, so the tests will trace the failure, even though we don't care. Useful for debugging.

```suggestion
	defer workertest.DirtyKill(c, w)
```

Reply:
There's no real point. We're doing a `CleanKill` at the end. This is only the first killer of the worker if we fail before, the cause of which will be pertinent output.

Reply:
> There's no real point.

Consistency is a strong counterargument to this.

Reply:
Indeed. I'd rather use the `workertest` methods anywhere we really need to wait for the worker. In all the places we don't (where it's basically garbage collection) just `Kill`. It means faster test feedback, like when you get "worker not stopping" for a missed mock expectation.

Reply:
Righto. I've done it.